confluent_platform/tests/compose/docker-compose.yaml

```diff
@@ -54,6 +55,7 @@ services:
     environment:
       SCHEMA_REGISTRY_HOST_NAME: schema-registry
       SCHEMA_REGISTRY_KAFKASTORE_CONNECTION_URL: 'zookeeper:2181'
+      SCHEMA_REGISTRY_LISTENERS: "http://0.0.0.0:8081"
```

Comment:
This was missing, it's used by the `connect` service:

```
CONNECT_VALUE_CONVERTER_SCHEMA_REGISTRY_URL: http://schema-registry:8081
```

This change doesn't impact tests, but it doesn't hurt to fix it.